Library/Formula/nsd.rb

```diff
@@ -24,12 +22,16 @@ def install
       --with-libevent=#{Formula["libevent"].opt_prefix}
     ]
 
-    args << '--enable-root-server' if build.with? 'root-server'
-    args << '--enable-bind8-stats' if build.with? 'bind8-stats'
-    args << '--enable-ratelimit' if build.with? 'ratelimit'
-    args << '--enable-zone-stats' if build.with? 'zone-stats'
+    args << "--enable-root-server" if build.with? "root-server"
+    args << "--enable-bind8-stats" if build.with? "bind8-stats"
+    args << "--enable-ratelimit" if build.with? "ratelimit"
+    args << "--enable-zone-stats" if build.with? "zone-stats"
 
     system "./configure", *args
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{sbin}/nsd", "-v"
```

Comment:
I'm not insisting; this test is a bit better than nothing but I just figured if you were doing cleanup you might be interested in doing an even better test. If you aren't: that's OK and I'll pull as-is.